cpp/src/io/json/read_json.cu

```diff
@@ -217,7 +227,14 @@ table_with_metadata read_json(host_span<std::unique_ptr<datasource>> sources,
                  "Multiple inputs are supported only for JSON Lines format");
   }
 
-  auto const buffer = get_record_range_raw_input(sources, reader_opts, stream);
+  auto buffer = get_record_range_raw_input(sources, reader_opts, stream);
+
+  // If input JSON buffer has single quotes and option to normalize single quotes is enabled,
+  // invoke pre-processing FST
+  if (reader_opts.is_enabled_normalize_single_quotes()) {
+    buffer = cudf::io::json::detail::normalize_single_quotes(
```

Comment:
why fully qualified name?

Reply:
Oops, just `normalize_single_quotes` is sufficient. Started with using the fully-qualified name in the test and ended up using it here as well.